librz/core/cprint.c

```diff
@@ -32,6 +32,23 @@ RZ_API RZ_OWN char *rz_core_print_string_c_cpp(RzCore *core) {
 	return rz_strbuf_drain(sb);
 }
 
+RZ_API void print_colored_help_tools(const char *option, const char *arg, const char *description, size_t maxOptionAndArgLength) {
```

Comment:
1. Rename it into `rz_core_print_colored_help_option()`, add Doxygen like for other `RZ_API` functions: https://github.com/rizinorg/rizin/blob/dev/DEVELOPERS.md#documentation